src/sync/installation.ts

```diff
@@ -94,7 +94,8 @@ export const updateJobStatus = async (
 	// Get a fresh subscription instance
 	const subscription = await Subscription.getSingleInstallation(
 		jiraHost,
-		installationId
+		installationId,
+		data.gitHubAppConfig?.gitHubAppId
```

Comment:
This isn't needed for the jira client but is needed when backfilling. Figured I was here editing this file so 🤷